src/hevm/src/EVM/Dapp.hs

```diff
@@ -62,6 +62,10 @@ makeLenses ''DappContext
 instance Show Test where
   show t = unpack $ extractSig t
 
+isSymbolic :: Test -> Bool
+isSymbolic (SymbolicTest _) = True
+isSymbolic _ = False
+
```

Comment:
I don't believe `isSymbolic` is used anywhere? Maybe it's not needed?

Reply:
nice catch :)